### PR TITLE
Fixed nested dir upload logic

### PR DESCRIPTION
**Pull request by mohami2000**
and updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,9 +118,6 @@ After setting up your credentials by any of these methods, you can run the integ
 ```sh
 # Run all tests
 hatch run integration-test
-
-# Run a single test file
-hatch run integration-test integration-test/test_<SOME_FILE>.py
```
Comment (Philmod) [inline README.md:123]: Did you remove on purpose?

**Review by Philmod — APPROVED** [review]
Thanks!

I'm assuming you tested locally?

File: README.md (hunk 1 above)
Comment (mohami2000) [reply README.md:123]: yes

Comment (rosbo) [reply README.md:123]: Why did you remove it?

**Pull request merged**